.github/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 [**Upgrade Guide**](https://intelowl.readthedocs.io/en/latest/Installation.html#update-to-the-most-recent-version)
 
+## [v3.0.0](https://github.com/intelowlproject/IntelOwl/releases/tag/v3.0.0)
+- **TLP Support**: Standardized threat-sharing using Traffic Light Protocol or `TLP`, thereby deprecating the use of booleans `force_privacy`, `disable_external_analyzers` and `private`. See [TLP Support](https://intelowl.readthedocs.io/en/master/Usage.html#tlp-support).
+- **New class of plugins called _Connectors_**
+  - Connectors are designed to run after every successful analysis which makes them suitable for automated threat-sharing. Built to support integration with other SIEM/SOAR projects specifically aimed at Threat Sharing Platforms. See [Available Connectors](https://intelowl.readthedocs.io/en/master/Usage.html#available-connectors).
```

Comment:
I like this paragraph about what "connectors" are. This should be duplicated to the `Usage.md` as well.